README.md

```diff
@@ -1,14 +1,260 @@
+
+### Public Classes
+* ['svckill'](#svckill-class): Main class used to set up a set of default
+services to ignore in an infrastructure.
+* ['svckill::ignore'](#svckillignore): Ensure a service will not be killed by
+svckill
+* ['svckill::ignore::collector'](#svckillignorecollector): Build default ignore
+file for svckill
+
+### `svckill` Type
+
+#### Parameters
+
+##### ignore
+  An array of services to never kill. Can also accept a regex.
+  * Valid Options: Array of Strings.
+
+##### ignorefiles
+  An array of files containing a list of services to ignore, one
+per line. Can also accept regexes in the file.
+  * Valid Options: Array of Strings.
+  * Default: `/usr/local/etc/svckill.ignore`
+
+##### verbose
+  If set, output all services that were affected by svckill.
+  * Valid Options: true or false.
+  * Default: `true`
+
+##### mode
+  If set to 'enforcing', will actually shut down and disable all
+services not listed in your manifests or the exclusion file. If set to
+'warning', will only report on what would happen without actually making the
+changes to the system.
+  * Valid Options: enforcing and warning.
+  * Default: `enforcing`.
+
+### `svckill` Class
+
+#### Parameters
+
+##### ignore
+  An array of services to never kill. Can also accept a regex.
+  * Valid Options: Array of Strings.
+  * Default: `[]`.
+
+##### ignore_defaults
+  An internal list of embedded services to never kill.
+  * Valid Options: Array of Strings.
+  * Default: `[]`.
+
+##### ignore_files
+  An array of files containing a list of services to ignore, one
+per line. Can also accept regexes in the file.
+  * Valid Options: Array of Strings.
+  * Default: `[]`.
+
+##### verbose
+  If set, output all services that were affected by svckill.
+  * Valid Options: true or false.
+  * Default: `true`
+
+##### mode
+  If set to 'enforcing', will actually shut down and disable all
+services not listed in your manifests or the exclusion file. If set to
+'warning', will only report on what would happen without actually making the
+changes to the system.
+  * Valid Options: enforcing and warning.
+  * Default:`enforcing`.
+
+### `svckill::ignore`
+
+#### Parameters
+
+##### name
+  Name of service to ignore with svckill.
+  * Valid Options: String.
+
+### `svckill::ignore::collector`
+
+#### Parameters
+
+##### default_ignore_file
+  Path to ignore file.
+  * Valid Options: Absolute Path.
+  * Default: `/usr/local/etc/svckill.ignore`.
 
-If you find any issues, they can be submitted to our [JIRA](https://simp-project.atlassian.net/).
+## Limitations
 
-Please read our [Contribution Guide](https://simp-project.atlassian.net/wiki/display/SD/Contributing+to+SIMP) and visit our [developer wiki](https://simp-project.atlassian.net/wiki/display/SD/SIMP+Development+Home).
+This module is only designed to work in RHEL or CentOS 6 and 7. Any other
+operating systems have not been tested and results cannot be guaranteed.
 
-## Work in Progress
+# Development
```

Comment:
To fit in with the other top-level TOC items, this should be a "`##`" level heading.